ConsentViewController/Classes/PrivacyManagerTab.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  PrivacyManagerTab.swift
+//  ConsentViewController
+//
+//  Created by Vilas on 28/12/20.
+//
+
+import Foundation
+
+/// Supported privacy manager tabs in PM
+public enum PrivacyManagerTab: String, CaseIterable {
+    /// having it empty string instructs the rendering app to use the defult tab of PM
```

Comment:
_or the pm tab specified in the "Show Options" action_

Reply:
Will update the comment